scripts/deploy.sh

```diff
@@ -133,6 +133,14 @@ openstack-ansible -i "localhost," patcher.yml
 
 # begin the openstack installation
 if [[ "${DEPLOY_OA}" == "yes" ]]; then
+
+  # This deploy script is also used for minor upgrades (within an openstack release)
+  # Some versions of liberty deploy pip lockdown to the repo server, in order for an
+  # upgrade to succeed the pip config must be removed so that repo builds have
+  # access to external repos.
+  # Issue tracking upstream fix: https://github.com/rcbops/rpc-openstack/issues/1028
+  ansible repo_all -m file -a 'name=/root/.pip state=absent' 2>/dev/null ||:
```

Comment:
I don't love using `:` instead of `true` for future readability reasons, but this is really a nit.